### DISCUSSION TITLE
Filter agents

### DISCUSSION THREAD

**Discussion by kgguru** [AzuraCast/AzuraCast#4969] [Ideas]
Would be great to have an agents black list similar to IP blacklist. I know that is possible in IceCast KH15.
It's large pain to kick off unpleasant hackers every second. 
Thanks.

**Comment by Liam024** [comment]
I still believe that you should set IP blacklists in the server firewall. Icecast only provides the streams, nothing more: https://github.com/AzuraCast/AzuraCast/issues/4939

Otherwise you can also create a feature request here: https://features.azuracast.com
